packages/kotti-ui/source/kotti-i18n/locales/de-DE.ts

```diff
@@ -27,6 +27,48 @@ export const deDE: KottiI18n.Messages = {
 		replyButton: 'Antworten',
 		replyPlaceholder: 'Antwort an',
 	},
+	KtFieldFileUpload: {
+		button: {
+			acceptPhoto: 'Foto verwenden',
+			cancel: 'Absagen',
+			nextCamera: 'Nächste Kamera',
+			rejectPhoto: 'Foto wiederholen',
+			retry: 'Wiederholen',
+			takePhoto: 'Foto machen',
+		},
+		error: {
+			notAllowed: 'Die Erlaubnis zur Verwendung der Kamera wurde verweigert',
+			notFound: 'Keine verfügbare Kamera gefunden',
+			notSupported: 'Keine kompatible Kamera verfügbar',
+		},
+		label: {
+			capture: 'Foto machen',
+			error: 'Error',
+			review: 'Überprüfen',
+			unknown: 'Unbekannt',
+		},
+		statusMsg: {
+			CANCELED: 'Abgebrochen',
+			ERROR: 'Hochladen fehlgeschlagen, bitte versuchen Sie es erneut',
+			FINISHED: 'Fertig',
+			NOT_STARTED: 'Nicht angefangen',
+			PROCESSING: 'Verarbeiten...',
+			UPLOADED: 'Hochgeladen',
+			UPLOADING: 'Hochladen...',
+			READY_TO_UPLOAD: 'Bereit zum Hochladen bei Einreichung',
+		},
+		text: {
+			clickToUpload: 'Zum Hochladen klicken',
+			dragAndDrop: 'oder ziehen und ablegen',
+			learnMore: 'Mehr erfahren',
+			max: 'max.',
```

Comment:
why is punctuated?

Reply:
is it the abbreviation of maximal?

Reply:
Yes, that's an abbreviation as indicated in the design